tests/ga/test_extension.py

```diff
@@ -1464,7 +1462,7 @@ def test__old_handler_reports_failure_on_disable_fail_on_update(self, patch_get_
 
             self.assertFalse(old_version_kwargs['is_success'], "The last call to report event should be for a failure")
 
-            self.assertTrue(old_version_kwargs['message'].startswith('[ExtensionError]'), "No error reported")
+            self.assertTrue('Error' in old_version_kwargs['message'], "No error reported")
```

Comment:
Too loose check? Does `LaunchCommandError` show up?

Reply:
I kind of wanted it to be loose, incase someone changes some implementation later on I dont want this test to be changed too. As long as there's error in the message I think its good enough for the test.